ci/jenkins/test-vm.sh

```diff
@@ -217,10 +273,12 @@ function run_e2e_vms {
     export GOCACHE=${WORKDIR}/.cache/go-build
     export PATH=$GOROOT/bin:$PATH
 
+    linux_vm_hostnames="${LIN_HOSTNAMESG[@]}"
```

Comment:
typo in the variable name: LIN_HOSTNAMESG

but I would suggest just change L281 to
```
-windowsVMs="${WIN_HOSTNAMES[*]}" -linuxVMs="${LIN_HOSTNAMES[*]}"
```
https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Arrays

> If the word is double-quoted, `${name[*]}` expands to a single word with the value of each array member separated by the first character of the IFS variable, and `${name[@]}` expands each element of name to a separate word.

Reply:
Added.